content/applications/hr/payroll.rst

```diff
@@ -91,32 +88,41 @@ form:
    entry remains 5.5 hours. If :guilabel:`Rounding` is set to :guilabel:`Half Day`, the entry is
    changed to 4 hours. If it is set to :guilabel:`Day`, it is changed to 8 hours.
 
-Working times
--------------
+- :guilabel:`Unpaid in Structures Types`: If the work entry is for unpaid work, specify which pay
+  structure the work entry applies to from the drop-down menu.
```

Comment:
These would be used for unpaid workers such as volunteers, interns, or people who might be working at the facility/company but being paid by someone else (like a visiting worker?) - not sure about that last one, but it's when you need to do a payslip entry to log their time but they are not being paid for whatever (legal) reason . Could be when you have a job that does unpaid training or something.

Reply:
I added that to clarify this- because you are right, it honestly doesn't make sense at first glance.  Odoo is usually for businesses, so why would you have timesheets for unpaid work in the payroll app =)